### PR TITLE
call response creation without call + call creation

### LINKED ISSUE #41: Member "on my way" mark
As a member I want to mark that I am on my way even without an alarm

### LINKED ISSUE #42: Call dispatch Member Based
As a member I want to get a call when enough members marked themself as "on my way".
As a department leader I want to set a threshold after which a call is dispatched automatically.

### PR DESCRIPTION
**Pull request by Integer93**
Closes #41 
Closes #42 

Member can now create a CallResponse without a call.
If there are >= 3 responses within 5 min, a call will be created and attached to them.

### CODE REVIEW

File: src/main/java/io/github/opendme/server/service/CallService.java [hunk 1]
```diff
@@ -23,12 +24,15 @@
 
 @Service
 public class CallService {
+    public static final Duration AUTO_DISPATCH_WINDOW = Duration.ofMinutes(5);
+    public static final int AUTO_DISPATCH_THRESHOLD = 3;
```
Comment (yannicklamprecht) [inline src/main/java/io/github/opendme/server/service/CallService.java]: Make both values configurable.

File: src/main/java/io/github/opendme/server/entity/CallResponseRepository.java [hunk 2]
```diff
@@ -2,5 +2,9 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.time.LocalDateTime;
+import java.util.List;
+
 public interface CallResponseRepository extends JpaRepository<CallResponse, Long> {
+    List<CallResponse> findAllByMember_Department_IdAndCreatedAtAfterAndCallNull(Long id, LocalDateTime createdAt);
```
Comment (yannicklamprecht) [inline src/main/java/io/github/opendme/server/entity/CallResponseRepository.java]: I assume the underscore is needed?

File: src/test/java/io/github/opendme/server/controller/CallControllerIT.java [hunk 3]
```diff
@@ -57,6 +59,10 @@ void setUp() {
         memberRepository.removeAllDepartments();
         departmentRepository.deleteAll();
         memberRepository.deleteAll();
+
```
Comment (yannicklamprecht) [inline src/test/java/io/github/opendme/server/controller/CallControllerIT.java]: Just auto wire the object mapper. There is a bean declared in TestGlobalBeans or how the class is called.

**Review by yannicklamprecht — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/opendme/server/entity/CallResponseRepository.java (hunk 2 above)
Comment (Integer93) [reply src/main/java/io/github/opendme/server/entity/CallResponseRepository.java]: Ah i didn't check that.
It also works without them.

File: src/main/java/io/github/opendme/server/service/CallService.java (hunk 1 above)
Comment (Integer93) [reply src/main/java/io/github/opendme/server/service/CallService.java]: I just used `@Value` in the service and put them in the profile. Hope that's enough.

**Review by yannicklamprecht — APPROVED** [review]

**Pull request merged**